plugins/modules/cloud/misc/proxmox.py

```diff
@@ -168,16 +154,15 @@
     version_added: '0.2.0'
   proxmox_default_behavior:
     description:
-      - Various module options used to have default values. This cause problems when
-        user expects different behavior from proxmox by default or fill options which cause
-        problems when they have been set.
-      - The default value is C(compatibility), which will ensure that the default values
-        are used when the values are not explicitly specified by the user.
-      - From community.general 4.0.0 on, the default value will switch to C(no_defaults). To avoid
-        deprecation warnings, please set I(proxmox_default_behavior) to an explicit
-        value.
+      - Since community.general 4.0.0, various options have no default values anymore.
+        These caused problems when user expects different behavior from proxmox by default or fill options
+        which caused problems when they have been set.
```

Comment:
```suggestion
      - As of community.general 4.0.0, various options no longer have default values.
        These default values caused problems when users expected different behavior from Proxmox 
        by default or filled options which caused problems when set.
```
A little more concise wording here makes sense to me.

Reply:
Thanks, I added both.